UefiCpuPkg/Include/Library/AmdSysCallLib.h

```diff
@@ -0,0 +1,59 @@
+/** @file
+  Provides function interface to perform syscall.
+
+Copyright (C) Microsoft Corporation.
+Copyright (C) 2025 Advanced Micro Devices, Inc. All rights reserved.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#pragma once

```

Comment:
Is there a reason not to use the normal include guard convention here?

Reply:
I updated the h file. How I submit a formal proposal?

Reply:
Any recommended format for rfc@edk2.groups.io, or just an e-mail explaining the pros and suggested spec change?